elixir-ts-mode.el

```diff
@@ -613,6 +613,12 @@ Return nil if NODE is not a defun node or doesn't have a name."
     (save-excursion
       (newline 1 t))))
 
+(defcustom elixir-ts-mode-hook '()
+  "Hook run after entering Elixir mode."
```

Comment:
"Hook run after entering `elixir-ts-mode'." its not Elixir mode.

Reply:
I used generated docstring, it uses the mode lighter